types/react-mentions/index.d.ts

```diff
@@ -101,7 +101,9 @@ export interface MentionProps {
 export interface MentionItem {
     display: string;
     id: string;
-    type: null;
```

Comment:
@k-funk I'm assuming this `type: null` is outdated and no longer needed. Arguably this is a breaking change though so I'll just mention it is worth mentioning to make sure it wasn't inadvertent. It's 4yo, I couldn't find any obvious use of it in docs, examples, or source, and has a bit of an odd definition anyway (and yes, I see I originally am to blame for that, but I can't remember why 🤦 ).
So I went ahead and approved it but you can confirm you did this explicitly by asking the bot to merge it.

Reply:
I found the same thing. Seems like a safe change to me. I'd actually be surprised if anyone was using this type (don't quote me. ever 😆).